src/main/java/com/minecolonies/entity/EntityCitizen.java

```diff
@@ -110,13 +98,13 @@
      * Divide experience by a factor to ensure more levels fit in an int.
      */
     private static final int EXP_DIVIDER = 10;
-    private static Field        navigatorField;
-    protected Status status = Status.IDLE;
-    private RenderBipedCitizen.Model modelId = RenderBipedCitizen.Model.SETTLER;
+    private static Field navigatorField;
+    protected Status                   status  = Status.IDLE;
+    private   RenderBipedCitizen.Model modelId = RenderBipedCitizen.Model.SETTLER;
     private String           renderMetadata;
     private ResourceLocation texture;
     private InventoryCitizen inventory;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Private field 'inventory' could be made final; it is only initialized in the declaration or constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3AImmutableField)